src/Targets/Microsoft.CSharp.Designer.targets

```diff
@@ -0,0 +1,23 @@
+<!--
+
+  This file contains C# Visual Studio and designer-related properties and items for C# projects.
+  
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <AppDesignerFolder Condition="'$(AppDesignerFolder)' == ''">Properties</AppDesignerFolder>
+  
```

Comment:
Could you please explain what happens when the condition `"'$(AppDesignerFolder)' == ''"` is true?